addons/sale/views/product_views.xml

```diff
@@ -130,4 +130,15 @@
         </field>
     </record>
 
+    <record id="view_category_property_form" model="ir.ui.view">
+        <field name="name">product.category.property.form.inherit</field>
+        <field name="model">product.category</field>
+        <field name="inherit_id" ref="product.product_category_form_view"/>
+        <field name="arch" type="xml">
+            <field name="property_account_expense_categ_id" position="after">
+                <field name="property_account_downpayment_categ_id"/>
```

Comment:
You need `groups="account.group_account_readonly"`

Reply:
it's already put on the group, so if i'm not mistaken no need to add it 
![image](https://github.com/user-attachments/assets/cf0fac95-2394-4086-913c-888e82fab0e6)